### PR TITLE
Add recipe for phyloacc

### PR DESCRIPTION
**Pull request by gwct**
This PR adds a recipe for [PhyloAcc](https://github.com/phyloacc/PhyloAcc), which is software for estimating substitution rates and rate shifts in conserved non-exonic regions.

The main code base is built and installed and the Python interface is copied to `$PREFIX/bin/` and its associated libraries to `$SP_DIR/phyloacc_lib/`.

### CODE REVIEW

**Comment by gwct** [comment]
@BiocondaBot please add label

File: recipes/phyloacc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - gsl
+    - openblas
+    - liblapack
+    - armadillo
+    - python >=3.6
```
Comment (dpryan79) [inline recipes/phyloacc/meta.yaml:36]: Please remove ">=3.6" here and from the "run:" section. Then under "build:" on line 23 put "skip: True  # [py2k]`.

**Review by dpryan79 — CHANGES_REQUESTED** [review]
See comment above

File: recipes/phyloacc/meta.yaml (hunk 1 above)
Comment (gwct) [reply recipes/phyloacc/meta.yaml]: Changes made in [ecf5e4a](https://github.com/bioconda/bioconda-recipes/pull/33464/commits/ecf5e4a4d8ecbf7001dae677576f34f73ae0222d). Thanks!

**Review by dpryan79 — APPROVED** [review]

**Pull request merged**